ReSharper.FSharp/test/data/parsing/Expr - Const - Measure - Seq 04.fs.gold

```diff
@@ -0,0 +1,35 @@
+﻿Language: PsiLanguageType:F#
+IFSharpImplFile
+  IAnonModuleDeclaration
+    IExpressionStatement
+      IChameleonExpression
+        ILiteralExpr
+          FSharpToken(type:INT32, text:123)
+          IUnitOfMeasureClause
+            FSharpTokenType+LessTokenElement(type:LESS, text:<)
+            ISeqMeasure
+              INamedMeasure
+                INamedTypeUsage
+                  ITypeReferenceName
+                    FSharpIdentifierToken(type:IDENTIFIER, text:cm)
+              Whitespace(type:WHITE_SPACE, text: ) spaces:" "
+              IVarMeasure
+                ITypeReferenceName
+                  ITypeParameterId
+                    FSharpTokenType+QuoteTokenElement(type:QUOTE, text:')
+                    FSharpIdentifierToken(type:IDENTIFIER, text:a)
+              Whitespace(type:WHITE_SPACE, text: ) spaces:" "
+              IPowerMeasure
+                INamedMeasure
+                  INamedTypeUsage
+                    ITypeReferenceName
+                      FSharpIdentifierToken(type:IDENTIFIER, text:kg)
+                FSharpIdentifierToken(type:SYMBOLIC_OP, text:^)
+                FSharpTokenType+LparenTokenElement(type:LPAREN, text:()
```

Comment:
Is this okay or would you like to introduce a structure for the rational cases?

Reply:
A proper structure would be beneficial. For example, features like 'Extend selection' would start working properly automatically.

Reply:
Ok, I tried to add more structure. Once we have https://github.com/dotnet/fsharp/pull/15614 in jetbrains/fsharp, this can get improved and simplified a lot, I guess.